### PR TITLE
Create sumo_entities_handler

### PR DESCRIPTION
**Pull request by elbrujohalcon**
It has to be built on trails and it has to provide the following endpoints:
- [x] `GET /[entities]` - return all entities
- [x] `POST /[entities]` - creates a new entity

### CODE REVIEW

File: elvis.config [hunk 1]
```diff
@@ -16,7 +16,6 @@
                   {elvis_style, nesting_level, #{level => 3}},
                   {elvis_style, god_modules, #{limit => 35}},
                   {elvis_style, no_if_expression},
-                  {elvis_style, invalid_dynamic_call},
```
Comment (igaray) [inline elvis.config:19]: Wouldn't adding the exception for the specific module be better?

Comment (elbrujohalcon) [inline elvis.config:19]: Almost all modules (at least those with actual code, i.e. not just behaviour definitions) in this repo will use dynamic calls… it's kinda the point here :)

File: src/sr_json.erl [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+-spec decode(iodata()) -> json().
+decode(Data) ->
+  try jiffy:decode(Data, [return_maps])
+  catch
+    _:_Error ->
+      throw(badjson)
+  end.
+
+-spec encode_date(calendar:datetime()) -> binary().
+encode_date({{Year, Month, Day}, {Hour, Min, Sec}}) ->
```
Comment (igaray) [inline src/sr_json.erl]: Would it make sense to put these in ktn_date ?

Comment (elbrujohalcon) [inline src/sr_json.erl]: The whole module, maybe?

File: src/sr_json.erl [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  try jiffy:decode(Data, [return_maps])
+  catch
+    _:_Error ->
+      throw(badjson)
+  end.
+
+-spec encode_date(calendar:datetime()) -> binary().
+encode_date(DateTime) -> iso8601:format(DateTime).
+
+-spec decode_date(binary()) -> calendar:datetime().
```
Comment (gadgetci) [inline src/sr_json.erl:38]: According to **Dialyzer**:

>  Invalid type specification for function sr_json:decode_date/1. The success typing is (nonempty_string()) -> {{non_neg_integer(),1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12,1..255},{byte(),byte(),byte()}}

**Pull request merged**